layouts/shortcodes/release-branches.html

```diff
@@ -0,0 +1,47 @@
+{{ range $releaseInfo := .Site.Data.releases.schedule.schedules }}
+{{- $version := printf "%.2f" $releaseInfo.release -}}
+{{- $dataIdVersion := replace ( printf "%.2f" $releaseInfo.release ) "." "-" | anchorize -}}
+
+<h3 id="{{ $dataIdVersion }}">{{ $version }}</h3>
+<p>
+    {{ printf (T "release_info_next_patch") $releaseInfo.next.release | markdownify }}
+</p>
+<p>
+    {{ $releaseInfo.next.note | markdownify }}
+</p>
+<p>
+    {{ printf (T "release_info_maintenance") $version | markdownify }} <span class="release-highlighted-date"><time datetime="{{ time.Format "2006-01-02" $releaseInfo.endOfLifeDate }}">{{ time.Format ( T "release_date_format") $releaseInfo.maintenanceModeStartDate }}</time></span> {{ T "release_info_eol" }} <span class="release-highlighted-date"><time datetime="{{ time.Format "2006-01-02" $releaseInfo.endOfLifeDate }}">{{ time.Format ( T "release_date_format") $releaseInfo.endOfLifeDate }}</time></span>
```

Comment:
This'll be easier to maintain and review if we split it over multiple lines.

Reply:
This assumes a grammatical structure of the sentence that we interpolate `release_info_maintenance` and `release_info_eol` into.

If you can see the issue, you're welcome to fix it. Otherwise, I'll try to make time to sketch out an alternative.

Reply:
I understand the issue here, considering I speak two other languages that don't conform to the grammatical structure in English. One solution I thought of was to have the whole sentence as a templated string but wasn't able to figure out how the time formatting would happen in this case.

I will try to do some more prototyping.

Reply:
> One solution I thought of was to have the whole sentence as a templated string

That's similar to how I'd do it. The overall call to `format` needs to take two HTML formatted elements - the dates - as part of their input. Possibly mark those formatted HTML elements  as `safeHTML`, if that seems appropriate.

Tip: you can pass a tuple or list around within a template; the things you pass don't have to be single items.

Reply:
I need to check out the `safeHTML` filter. The change that I pushed sometime back resolves the main issue.

Reply:
@sftim -- check out https://github.com/kubernetes/website/pull/36609/commits/24c6f0b84684b3706c298fc7e40a9c9bff47ed15. this should be the best way I feel.